components/closebutton/metadata/closebutton.yml

```diff
@@ -28,66 +22,6 @@ examples:
         <div class="spectrum-Examples-item">
           <h4 class="spectrum-Heading spectrum-Heading--sizeXS spectrum-Examples-itemHeading">S</h4>
 
-          <div class="spectrum-Examples-itemGroup">
-            <button class="spectrum-CloseButton spectrum-CloseButton--sizeS">
-              <svg class="spectrum-CloseButton-UIIcon spectrum-Icon spectrum-UIIcon-Cross75" focusable="false" aria-hidden="true">
-                <use xlink:href="#spectrum-css-icon-Cross75" />
-              </svg>
-            </button>
-          </div>
-        </div>
-        <div class="spectrum-Examples-item">
-          <h4 class="spectrum-Heading spectrum-Heading--sizeXS spectrum-Examples-itemHeading">M (default)</h4>
-
-          <div class="spectrum-Examples-itemGroup">
-            <button class="spectrum-CloseButton spectrum-CloseButton--sizeM">
-              <svg class="spectrum-CloseButton-UIIcon spectrum-Icon spectrum-UIIcon-Cross100" focusable="false" aria-hidden="true">
-                <use xlink:href="#spectrum-css-icon-Cross100" />
-              </svg>
-            </button>
-          </div>
-        </div>
-
-        <div class="spectrum-Examples-item">
-          <h4 class="spectrum-Heading spectrum-Heading--sizeXS spectrum-Examples-itemHeading">L</h4>
-
-          <div class="spectrum-Examples-itemGroup">
-            <button class="spectrum-CloseButton spectrum-CloseButton--sizeL">
-              <svg class="spectrum-CloseButton-UIIcon spectrum-Icon spectrum-UIIcon-Cross200" focusable="false" aria-hidden="true">
-                <use xlink:href="#spectrum-css-icon-Cross200" />
-              </svg>
-            </button>
-          </div>
-        </div>
-
-        <div class="spectrum-Examples-item">
-          <h4 class="spectrum-Heading spectrum-Heading--sizeXS spectrum-Examples-itemHeading">XL</h4>
-
-          <div class="spectrum-Examples-itemGroup">
-            <button class="spectrum-CloseButton spectrum-CloseButton--sizeXL">
-              <svg class="spectrum-CloseButton-UIIcon spectrum-Icon spectrum-UIIcon-Cross300" focusable="false" aria-hidden="true">
-                <use xlink:href="#spectrum-css-icon-Cross300" />
-              </svg>
-            </button>
-          </div>
-        </div>
-      </div>
-  - id: closebutton-largeicon
-    name: Icon Size - Large
```

Comment:
Per[ the changelog](https://www.figma.com/file/M9P2s2IDhGWZuNyvJtXrx3/S2-foundations-and-components-changelog?type=design&node-id=740-1679&mode=design&t=4cIhoO8rTWmFDwRy-0), icon size goes away in S2